### PR TITLE
[api-polish] Renaming api's 

### PR DESCRIPTION
**Pull request by arnavrneo**
# What does this PR do?

This PR fixes the following issues:
- #1247 
  - `BinaryPenaltyReducedFocalXent` is made consistent with Keras by changing it to `BinaryPenaltyReducedFocalCrossEntropy`

- #1248 and #1249
  - `global_random_scaling.py` has been updated by renaming all `scaling_factor_*` to `*_factor` and `same_scaling_xyz` to `preserve_aspect_ratio`.
 
- #1250 
  - The error messages have been improved in `frustum_random_droppping_points.py`. They now show the error along with the user-entered value.

## Before submitting 
- ✅  This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case). 
- ✅  Did you read the [contributor guideline](https://github.com/keras-team/keras-cv/blob/master/.github/CONTRIBUTING.md), Pull Request section? 
- [ ] Was this discussed/approved via a Github issue? Please add a link to it if that's the case. 
- [ ] Did you write any new necessary tests? 
- [ ] If this adds a new model, can you run a few training steps on TPU in Colab to ensure that no XLA incompatible OP are used?

### CODE REVIEW

**Comment by tanzhenyu** [comment]
Please have a LGTM with @lengzq before merging it, thanks!

File: keras_cv/layers/preprocessing3d/global_random_scaling.py
Comment (LukeWood) [inline keras_cv/layers/preprocessing3d/global_random_scaling.py:57]: @lengzq is this an accurate description of what this parameter does?

File: keras_cv/losses/penalty_reduced_focal_loss.py
Comment (LukeWood) [inline keras_cv/losses/penalty_reduced_focal_loss.py:21]: @tanzhenyu @lengzq Can either confirm that Xent is CrossEntropy?  If so this PR LGTM

**Comment by LukeWood** [comment]
Thanks @arnavrneo !

**Comment by arnavrneo** [comment]
@LukeWood Happy to contribute! 

I have one doubt. I see that the test codes are failing. Do I also need to update the api in the respective `*_test` files?

**Comment by LukeWood** [comment]
> @LukeWood Happy to contribute!
> 
> I have one doubt. I see that the test codes are failing. Do I also need to update the api in the respective `*_test` files?

Yes

**Comment by arnavrneo** [comment]
@LukeWood I've updated the api in `*_test` files also. Kindly review the PR again.

**Review by LukeWood — APPROVED** [review]
Looks good

**Comment by LukeWood** [comment]
@arnavrneo looks like there are some merge conflicts - can we handle these by rebasing?

File: keras_cv/layers/preprocessing3d/frustum_random_dropping_points.py
Comment (LukeWood) [inline keras_cv/layers/preprocessing3d/frustum_random_dropping_points.py:57]: nit:

```
            raise ValueError(f"r_distance must be >=0, but got r_distance={r_distance}")
```

File: keras_cv/layers/preprocessing3d/frustum_random_dropping_points.py
Comment (LukeWood) [inline keras_cv/layers/preprocessing3d/frustum_random_dropping_points.py:59]:
```
            raise ValueError(f"theta_width must be >=0, but got theta_width={theta_width}")
```

File: keras_cv/layers/preprocessing3d/frustum_random_dropping_points.py
Comment (LukeWood) [inline keras_cv/layers/preprocessing3d/frustum_random_dropping_points.py:61]: (same as above)

File: keras_cv/layers/preprocessing3d/frustum_random_dropping_points.py
Comment (LukeWood) [inline keras_cv/layers/preprocessing3d/frustum_random_dropping_points.py:66]: (same as above)

File: keras_cv/layers/preprocessing3d/frustum_random_dropping_points.py
Comment (LukeWood) [inline keras_cv/layers/preprocessing3d/frustum_random_dropping_points.py:64]: can drop_rate be negative?  I think it must be 0 < x < 1, right?

Comment (arnavrneo) [reply keras_cv/layers/preprocessing3d/frustum_random_dropping_points.py]: I think you're right, it should not be negative. I'm no expert at this but I guess as it represents the fraction of units to drop, it should be between 0 and 1. 
Besides the `Dropout` layer of `Keras` also needs a rate between 0 and 1, so.

**Comment by arnavrneo** [comment]
@LukeWood I've rebased it, I hope there should not be any merge conflicts now. 
The error messages have also been updated. Could you review it again?

**Comment by LukeWood** [comment]
@arnavrneo  thanks for the help - 

any chance you can fix the failing tests?  Thanks!

File: keras_cv/losses/penalty_reduced_focal_loss.py
Comment (lengzq) [reply keras_cv/losses/penalty_reduced_focal_loss.py:21]: CrossEntropy

**Review by lengzq — APPROVED** [review]

File: keras_cv/layers/preprocessing_3d/frustum_random_dropping_points.py
Comment (ianstenbit) [inline keras_cv/layers/preprocessing_3d/frustum_random_dropping_points.py:57]: nit: typo (s/r_distace/r_distance)

**Review by ianstenbit — APPROVED** [review]
Thank you! Looks good after the minor comment I left :)

File: keras_cv/layers/preprocessing_3d/frustum_random_dropping_points.py
Comment (arnavrneo) [reply keras_cv/layers/preprocessing_3d/frustum_random_dropping_points.py]: oops, the typo! I've fixed it now :)

**Comment by ianstenbit** [comment]
@arnavrneo could you please fix the linter error? Then I can merge :)

**Comment by arnavrneo** [comment]
@ianstenbit 

I've solved the linter error that was causing the trouble  
First, I formatted the files with `black` and ran `pytest` and it showed me the linter error (which I've sorted out) but now when I run `pytest` again, its giving me the error below:

### Error
```py
_______________________________________________________ ERROR collecting keras_cv/layers/preprocessing_3d/global_random_scaling_test.py _______________________________________________________
keras_cv/layers/preprocessing_3d/global_random_scaling_test.py:25: in <module>
    class GlobalRandomScaling(base_augmentation_layer_3d.BaseAugmentationLayer3D):
/usr/local/lib/python3.8/dist-packages/keras/saving/object_registration.py:148: in decorator
    raise ValueError(
E   ValueError: keras_cv>GlobalRandomScaling has already been registered to <class 'keras_cv.layers.preprocessing_3d.global_random_scaling.GlobalRandomScaling'>
=================================================================================== short test summary info ===================================================================================
ERROR keras_cv/layers/preprocessing_3d/global_random_scaling_test.py - ValueError: keras_cv>GlobalRandomScaling has already been registered to <class 'keras_cv.layers.preprocessing_3d.global_random_scaling.GlobalRandomScaling'>
```
If I ignore `global_random_scaling_test.py` for `pytest` checkup, everything's else fine. Can you check this out?

**Comment by ianstenbit** [comment]
@arnavrneo It looks like you've replaced the contents of the file `keras_cv/layers/preprocessing_3d/global_random_scaling_test.py` with the code from `global_random_scaling.py`.

**Comment by arnavrneo** [comment]
@ianstenbit You're right, I had messed up the two files :) My bad! Thanks for the help. I've ran the tests on my end and now everythings fine.

**Pull request merged**
